node/src/components/chainspec_loader/chainspec.rs

```diff
@@ -316,6 +324,8 @@ impl GenesisConfig {
         let highway_config = HighwayConfig::random(rng);
         let unbonding_delay = rng.gen();
         let wasmless_transfer_cost = rng.gen();
+        let initial_era_id = rng.gen();
+        let initial_block_height = rng.gen();
```

Comment:
A question: do we maybe want a more sophisticated method of randomly generating these values? Like `None` with some probability, and `Some(x)` with randomly chosen `x` with another probability?

Reply:
I don't think it matters if we don't do any type of chainspec validation.